mergebot_backend/merge.py

```diff
@@ -200,12 +205,26 @@ def run(self):
                 info="Requested by {user} at {time}.".format(
                     user=pr.metadata['asf_id'], time=pr.metadata['created']))
             try:
-                pr.post_info(
-                    'MergeBot starting work on PR#{pr_num}.'.format(
-                        pr_num=pr_num),
-                    self.merge_logger)
+                pr.post_commit_status(
+                    state=github_helper.COMMIT_STATE_PENDING,
+                    url=MERGEBOT_ITEM_URL.format(
+                        name=self.config.name, number=pr_num),
+                    description='Started Work',
```

Comment:
This indicates that MergeBot has begun work on this PR. I could add additional commit status updates which show when the status changes in intermediary ways, if that's something you think would be useful.